DigitalLearningSolutions.Web/Styles/trackingSystem/progress.scss

```diff
@@ -0,0 +1,7 @@
+@import "~nhsuk-frontend/packages/core/all";
+
+.tutorial-progress-table-headers {

```

Comment:
This should probably be an id rather than a class if we're being picky - it's referring to a specific element on the page rather than elements matching an abstract description.